rules/macos/credential_access_access_to_browser_credentials_procargs.toml

```diff
@@ -20,7 +20,9 @@ note = """## Setup
 
 If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
 """
-references = ["https://securelist.com/calisto-trojan-for-macos/86543/"]
+references = [
+  "https://securelist.com/calisto-trojan-for-macos/86543/"
+  ]
```

Comment:
Was there a Security Labs link you wanted to put here?

Reply:
It was in there, but removed since this is a macOS rule and not a Windows rule so the blog did not apply. The only change now is the formatting of the list.